tracker/src/main/java/org/matomo/sdk/extra/DownloadTracker.java

```diff
@@ -88,12 +85,6 @@ public String buildExtraIdentifier() {
             }
         }
 
-        /**
-         * Custom exta identifier. Supply your own \o/.
-         */
-        abstract class Custom implements Extra {
-        }
-
```

Comment:
I agree, this is kind of superfluos, but it's part of the public API and some devs might have extended this.

Reply:
Keeping this unused stuff is a kind of collect a lot waste in your base ground.

Reply:
We can remove it, but then with a major version bump and a little note about migration in the changelog (e.g. don't extend `Custom`, just implement `Extra`).